src/main/java/com/DoctorList.java

```diff
@@ -8,19 +8,21 @@
 
 	private final Map<Integer, Doctor> map = new TreeMap<>();
 
+	private boolean loopCondition = true;
+
 	public void choice() {
-		while (true) {
-			System.out.println("What do you want to do?");
-			System.out.println("1.Add the doctor.");
-			System.out.println("2.Remove the doctor.");
-			System.out.println("3.Show list of doctors.");
-			System.out.println("4.Exit ");
-			System.out.println("");
-			System.out.println("Choose number from 1 to 4: ");
+		while (loopCondition == true) {
+			StringBuilder printMainMessage = new StringBuilder();
```

Comment:
It's not a method, therefore you should use nouns to name variables.
Please rename it to mainMessage.